### ISSUE TITLE
ABINet deployment

### ISSUE THREAD

**Issue by Hegelim** [open-mmlab/mmdeploy#689]
Thanks for the response here https://github.com/open-mmlab/mmocr/issues/1124

1. just to confirm, ABINet is currently not supported to be converted to tensorRT using MMDeploy, right?
2. if possible, could you point out some documentation/resources if we want to develop the feature to convert ABINet pretrained model to tensorRT? What should we do first? What are some files we need? 

Thanks

**Comment by lvhan028** [comment]
For the first question, mmdeploy hasn't supported it yet
For the second question, @AllentDan could you please give some suggestions?

**Comment by AllentDan** [comment]
Well, our deployment routine follows PyTorch->ONNX->TensorRT.
Tips are:
1. build and enjoy MMDeploy first though [get_started](https://github.com/open-mmlab/mmdeploy/blob/master/docs/en/get_started.md). It would be better if you convert an OCR model to TensorRT and visualize the inference results through `tools/deploy.py`.
2. try converting the ABINet PyTorch model to ONNX. In this step, you may refer to doc [here](https://github.com/open-mmlab/mmdeploy/blob/master/docs/en/06-developer-guide/support_new_model.md).
3. try converting ONNX model to TensorRT engine. You may debug the model until the visualization result looks okay.

In summary, all you need to do is to install MMDeploy and run `toos/deploy.py`. In the process, if you got any error and have no idea how to fix it, it is okay to post it here. We will give suggestions at our best.

**Comment by AllentDan** [comment]
Closing it for no activity for a long time, feel free to reopen it if it is still an issue for you.

**Comment by qiyueliuhuo** [comment]
@Hegelim  I want to deploy ABINet model using MMDeploy. Have you successfully deployed the model?
